tests/conftest.py

```diff
@@ -120,13 +120,45 @@ def git_repository(fake_repository: Path, git_exe: Optional[str]) -> Path:
     build_dir.mkdir()
     (build_dir / "hello.py").touch()
 
-    subprocess.run([GIT_EXE, "init", str(fake_repository)])
-    subprocess.run([GIT_EXE, "add", str(fake_repository)])
-    subprocess.run([GIT_EXE, "commit", "-m", "initial"])
+    subprocess.run([git_exe, "init", str(fake_repository)])
```

Comment:
Out of curiosity, why is this lower case now?

Reply:
I would imagine to use the `git_exe` fixture instead of the global variable. They are practically identical, but using a local variable instead of a global variable is a little bit nicer.